mapdata/WarcraftLegacies/Doodads/1024_10240.json

```diff
@@ -295,7 +295,7 @@
       "Y": 10176,
       "Z": 281.5
     },
-    "Rotation": -1.4017224,
+    "Rotation": 4.881463,
```

Comment:
Revert changes to Doodads and Units folders please (assuming they're not meant to be here).